e2e-tests/web/test/specs/navigateToNewLocation.e2e.ts

```diff
@@ -0,0 +1,28 @@
+import { URL } from 'url'
+
+import { Routes } from '../../../shared/constants'
+import DashboardPage from '../pageobjects/dashboard.page'
+import LandingPage from '../pageobjects/landing.page'
+
+describe('navigateToOtherLocation', () => {
+  it('should open an other location', async () => {
+    const landingPath = Routes.landing
+    await DashboardPage.open()
+
+    const locationIcon = await DashboardPage.locationIcon
+    await locationIcon.click()
+
+    const landingUrl = await browser.getUrl()
+    const parsedLandingUrl = new URL(landingUrl)
+    expect(parsedLandingUrl.pathname).toContain(landingPath)
+
+    const filteredCity = await LandingPage.city('Stadt Augsburg')
+    expect(filteredCity).toBeDefined()
```

Comment:
only :art: : superficient, no need to test this, this will fail on next step with "trying to call click on an undefined object" anyways

Reply:
yeah, but on a failing test it is much nicer to get the feedback `filteredCity should be defined but is not`, instead of `undefined is not a object or something`